changelog/TEMPLATE

```diff
@@ -1,11 +1,21 @@
-Bugfix: Fix behavior for foobar (in present tense)
+# The first line must start with Bugfix:, Enhancement: or Change:,
+# including the colon. Use present use. Remove lines starting with '#'
+# from this template.
+Enhancement: Allow custom bar in the foo command
 
-We've fixed the behavior for foobar, a long-standing annoyance for restic
-users.
+# Describe the problem in the past tense, the new behavior in the present
+# tense. Mention the affected commands, backends, operating systems, etc.
+# Focus on user-facing behavior, not the implementation.
```

Comment:
I think we should probably add this sentence at the end of this paragraph: `If the text becomes long, split the past/old and the present/new parts into two paragraphs.`

Reply:
That enforces a rule that I already know I'm going to try and break. When the first paragraph is too long, I would rather summarize both the problem and the solution in the first paragraph, then add more paragraphs with details. Maybe leave this to the contributor? Your call, obviously.

Reply:
I don't agree with how you'd do it, but if it doesn't work out in the end we'll just comment on it at that point, so I don't think this is important enough to argue about.